src/components/forms/SignUpForm/index.tsx

```diff
@@ -12,8 +15,15 @@ function SignUpForm() {
     getValues,
     formState: { errors }
   } = useForm<User>();
-  // eslint-disable-next-line no-console
-  const onSubmit = () => console.log(getValues());
+  const [errorMsg, setErrorMsg] = useState('');
+  const { isLoading, isError, mutate } = useMutation((data: User) => signUp(data), {
+    // eslint-disable-next-line no-console
+    onSuccess: res => console.log('res', res),
+    onError: () => {
+      setErrorMsg(i18next.t('FormValidations:network'));
+    }
+  });
+  const onSubmit = () => mutate(getValues());
```

Comment:
Same as the `onSuccess` handler, we only want to execute a block, not to return anything

```suggestion
  const onSubmit = () => {
      mutate(getValues());
  };
```

Reply:
👍 Solved!